README.md

```diff
@@ -13,10 +13,28 @@ Chris S Magnano, Anthony Gitter.
 
 ## Dependencies
 
-Pathway parameter advising was written and tested using Python 3.6 and requires the packages `networkx` and `numpy`.
-The `runPPA.sh` and `setupPGD.sh` scripts require git.
+Pathway parameter advising was written and tested using Python 3.6 and requires the packages `networkx`, `numpy`, and `requests`.
+
+Graphlet decomposition is performed using the PGD library.
+The PGD library can be installed from its [github repository](https://github.com/nkahmed/pgd) and complied using `make`.
+The script `setupPGD.sh` requires git, and has been provided to aid in installing the PGD library.
+However, it is not guaranteed to work and has only been tested on Ubuntu 18.04.
+
 These scripts have only been tested in a Linux environment initially.
 
+## Installation
+To download pathway parameter advising using pip, run the command:
+> 'pip download --no-deps --no-binary :all: pathwayParameterAdvising`
+
+And untar the resulting archive file. 
+
+Pathway parameter advising can also be downloaded from [the pathway parameter advising github](https://github.com/gitter-lab/pathway-parameter-advising/tree/reactomeManagement). 
+
+From inside `pathway-parameter-advising` directory, run
+> `python setup.py`
```

Comment:
I got an error at this line
```
$ python setup.py
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: no commands supplied
```

Should the command be `python setup.py install` or ` pip install . `?  I believe the pip version is often recommended.

Reply:
I changed the readme to instruct users to use `python setup.py install`. `pip install .` gave me an issue where I think it would want me to also download the `.egg` file to install properly (which is an option, but I left it out for now).